addons/website_discuss_room/static/src/scss/chat_room.scss

```diff
@@ -0,0 +1,16 @@
+.o_wdiscuss_room_modal .modal-dialog {
+    max-width: 100vw;
+    max-height: 100vh;
+    margin: auto;
```

Comment:
These part could be replaced by using `vh-100 m-auto` in the template. Most of the time, it's better to use bootsrap classes when possible